### ISSUE TITLE
ember-try scenario ember-default-with-jquery fails when ember-source is 4.0.0

### ISSUE THREAD

**Issue by SergeAstapov** [ember-cli/ember-cli#9701]
Ember CLI addon blueprint has `ember-try` scenario `ember-default-with-jquery` that uses `ember-source` version of the addon's `package.json`.

If `ember-source` is 4.0.0 then scenario in CI fails with error:
```
=== Scenario: ember-default-with-jquery ========================================

Setting the `jquery-integration` optional feature flag to `true` was deprecated in Ember 3.x and removed in Ember 4.0.0. You must add the `@ember/optional-features` addon and set this feature to `false`.

For more information, see the deprecation guide: https://deprecations.emberjs.com/v3.x/#toc_optional-feature-jquery-integration


Result: false
---
```

Currently scenario is setup like so:
```js
{
  name: 'ember-default-with-jquery',
  env: {
    EMBER_OPTIONAL_FEATURES: JSON.stringify({
      'jquery-integration': true,
    }),
  },
  npm: {
    devDependencies: {
      '@ember/jquery': '^1.1.0',
    },
  },
}
```

Thinking of options, this scenarios could set be renamed to `ember-classic-with-jquery` with `'ember-source': '~3.28.0'`.

**Comment by SergeAstapov** [comment]
For the context, actual error with Ember CLI 4.0.0 is

```
=== Scenario: ember-default-with-jquery ========================================

Setting the `jquery-integration` optional feature flag to `true` was deprecated in Ember 3.x and removed in Ember 4.0.0. You must add the `@ember/optional-features` addon and set this feature to `false`.

For more information, see the deprecation guide: https://deprecations.emberjs.com/v3.x/#toc_optional-feature-jquery-integration

Result: false
```

**Comment by BnitoBzh** [comment]
I'm agree  with @SergeAstapov 
> Thinking of options, this scenarios could set be renamed to ember-classic-with-jquery with 'ember-source': '~3.28.0'.

**Comment by SergeAstapov** [comment]
This was fixed by #9737
